internal/test/storage.go

```diff
@@ -94,29 +94,46 @@ func (m *MockKeyValueStore) Delete(key storage.Key) error {
 }
 
 // List returns either a list of keys for which the MockKeyValueStore has values or an error.
-func (m *MockKeyValueStore) List(startingKey storage.Key, limit storage.Limit) (storage.Keys, error) {
+func (m *MockKeyValueStore) List(opts storage.ListOptions) (storage.Items, storage.More, error) {
+
 	m.ListCalled++
 	keys := storage.Keys{}
 	keySlice := mapIntoSlice(m.Data)
 	started := false
 
-	if startingKey == nil {
+	listItems := storage.Items{}
+
+	if opts.Start == nil {
 		started = true
 	}
 	for _, key := range keySlice {
-		if !started && key == string(startingKey) {
+		if !started && key == string(opts.Start) {
```

Comment:
so, if someone provides a StartKey, it should return every key after it, not including it? it's a noninclusive start?